cwp/cwp-working-groups.md

```diff
@@ -1,231 +1,233 @@
-
-Plans:
-
-Draft WG white paper: [ShareLaTeX](https://www.sharelatex.com/project/595500273c5204ff35dfdcf9)
-
-<!-- ======================================================= -->
-
-### Physics Generators WG ###
-Charge: [Google Doc](https://docs.google.com/document/d/1_-YzxFmwehLbjVRFzkKtJ3Nb4T0fN15VKYJIwRpKdVA/edit)
-
-Google group: none yet 
-
-Plans:
-
-Draft WG white paper: [Overleaf](https://www.overleaf.com/read/wyyybnvxyfyn)
-
-<!-- ======================================================= -->
-
-### Math Libraries WG ###
-Charge: [Google Doc](https://docs.google.com/document/d/1cYs5f21d1olumslSfjOH0XjYR7aeM6QZMrFVHFHDQEk/edit) 
-
-Google group: hsf-cwp-math-libaries@googlegroups.com [(link to subscribe)](https://groups.google.com/forum/#!forum/hsf-cwp-math-libaries) 
-
-Plans:
-
-<!-- ======================================================= -->
-
-### Data Acquisition Software WG ###
-Charge: [Google Doc](https://docs.google.com/document/d/1BYeWbDLyxG-CUV70aKR-UJubwBdChVrb9lxI1Rk43sY/edit)
- 
-Status: not yet active
-
-Google group: none yet
-
-Plans:
-
-<!-- ======================================================= -->
-
-### Various Aspects of Technical Evolution (Software Tools, Hardware, Networking) WG ###
-Charge: [Google Doc](https://docs.google.com/document/d/1bOv5kDzmcg0eLSy7PZjjzE5OYSxaLAfaAxAI-IgKaac/edit)
-
-Status: not yet active
-
-Google group: none yet
-
-Plans:
-
-<!-- ======================================================= -->
-
-### Monitoring WG ###
-Charge: [Google Doc](https://docs.google.com/document/d/1sgTrDSwiYthpnE2sztaaByMkJyUXzUy28bUbs9VLI1w/edit)
-
-Status: not yet active
-
-Google group: none yet
-
-Plans:
-
-<!-- ======================================================= -->
-
-### Security and Access Control WG ###
-[Google Doc](https://docs.google.com/document/d/17xVZv8GJIRqCdFQeH2Ec7J8AmC7HokuRbyNR2K3SryY/edit)
-
-Google group: None yet
-
-Plans:
-
-
-
- 
+---
+title: "Community White Paper Working Groups"
+author: "Peter Elmer"
+layout: default
+---
+
+## Community White Paper (CWP) Working Groups
+
+  During October, 2016, the process of forming CWP working groups has begun. We are letting the working groups self-organize with a bit of help from the HEP experiment software/computing coordinators and the HSF startup team. The first step for each working group is to formulate a charge to describe the challenges in that area and questions which need to be answered to produce a roadmap for the CWP. 
+
+  The working group charges should be world-visible for reading. If you would
+like to edit them, you will need to join the [hsf-community-white-paper](https://groups.google.com/forum/#!forum/hsf-community-white-paper) Google group and
+login as your google user. If you would like organize an additional working group, please post to the [hsf-community-white-paper](https://groups.google.com/forum/#!forum/hsf-community-white-paper) Google group.
```

Comment:
adding groups looks outdated to me

Reply:
Right - last commit generally revamped the phrasing there to reflect better the stage of the process we are in.